lib/ReactViews/ReactViewHelpers/highlightKeyword.jsx

```diff
@@ -1,6 +1,11 @@
 export default function highlightKeyword(searchResult, keywordToHighlight) {
   if (!keywordToHighlight) return searchResult;
-  const parts = searchResult.split(new RegExp(`(${keywordToHighlight})`, "gi"));
+  const escapedKeyword = keywordToHighlight.replace(
+    /[.*+?^${}()|[\]\\]/g,
+    "\\$&"
+  );
```

Comment:
BTW, there is a new [RegExp.escape](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/escape), although it is pretty new.